services/ui-src/src/forms/mcpar/cpc/cpc.json

```
@@ -11,8 +15,129 @@
         "id": "cpc-1",
         "type": "text",
         "props": {
-          "label": "Temporary placeholder field",
-          "disabled": true
+          "label": "C1.I.1 Program contract",
+          "hint": "Enter the title and date of the contract between the state and plans participating in the managed care program."
+        }
+      },
+      {
+        "id": "cpc-1-b",
```

Hmm... we don't really have a standard format for this yet, but I think it would make sense for fields like this to do something like:
```suggestion
        "id": "cpc-1-sub",
```